src/components/tables/action-buttons.tsx

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from "next/navigation";
+import { Button } from "../ui/button";
+import { FileEdit, Trash2 } from "lucide-react";
+import { PageRoutes } from "@/constants/page-routes";
+
+interface Props {
+    row: any
+    route: string
+}
+const ActionButtons = ({ row, route }: Props) => {
+    const router = useRouter();
+    const data = row.original;
+
+    return (
+        <div className='flex items-center gap-2'>
+            <Button type="button" variant={"link"} onClick={() => router.push(`${route}/${data.id}`)}><FileEdit size={17} color='black' /></Button>
```

Comment:
also we don't need router onclick here. Let's wrap it in a Link so that it pre-fetches the data to improve experience